src/main/java/org/zalando/nakadi/service/subscription/state/ClosingState.java

```diff
@@ -75,7 +76,7 @@ public void onEnter() {
 
     private void onTopologyChanged() {
         if (topologyListener == null) {
-            TracingService.logErrorInSpan(TracingService.activateSpan(getContext().getCurrentSpan(), false),
+            TracingService.logErrorInSpan(getContext().getCurrentSpan(),
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs